docs/2.9.0/docs/query/pqs-user-guide.rst

```diff
@@ -4,13 +4,6 @@
 Participant Query Store User Guide
 ##################################
 
-Docker image
-************
-
-To get the Participant Query Store (PQS) Docker image, run the following command::
-
-    docker pull digitalasset-docker.jfrog.io/participant-query-store:0.1.0
-
```

Comment:
This seems unrelated.

Reply:
Someone put it there but had it as the first item in the page. I moved it after the PQS intro so that a user could at least understand why they might want to get the Docker image.

Reply:
That someone was me. I put it there because it's a recurring question from people who already know what PQS and know they want it.

Reply:
Yes.  I just moved it below the introduction.  Is that ok?

Reply:
With the mode the Docker information is close enough to the top to be seen.  I do think the introduction is more important than the Docker information because the first question a potential user has is: "Why am I interested in this?"  The second question is "How can I get it?"

Reply:
There's definitely a tradeoff, but consider that the scenario you describe only applies to each user at most once. They may already know they need it for any number of reasons, among which: they've already read the page in the past, their team is using it and they're just joining the project, or their DA contact told them this is the tool they need for the problem they have.